build.gradle

```diff
@@ -22,11 +22,11 @@ gretty {
     servletContainer = "jetty9.4"
 }
 
-// The following pnpmEnable = true is not needed as pnpm is used by default,
+// The following pnpmEnable = false is not needed as pnpm is used by default,
```

Comment:
"The following pnpmEnable = false is not needed as pnpm is used by default"

This sounds contradictory. So npm was the default, right?

Reply:
npm is the current default. didn't read to the end. Fixed